recipe/meta.yaml

```diff
@@ -74,24 +75,22 @@ test:
     - pip
     - pytest >=6
     - pytest-cov
-    - pytest-asyncio >0.11, !=0.22.0, <0.23.0
+    - pytest-asyncio >0.11,!=0.22.0,<0.23.0
     - pytest-xdist
     - pytest-env
     - pytest-benchmark
     - aiohttp
     - nest-asyncio
-    - websockets
+    - websockets <14
     - ipykernel
     - joblib
 
 about:
   home: https://libertem.github.io/LiberTEM/
   summary: Open pixelated STEM framework
-  license: GPL-3.0-or-later AND MIT
```

Comment:
For this point release, we still have GPL right?

Reply:
Good point, I'll roll it back :see_no_evil: